### PR TITLE
Polish what is section

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
## fixes KILTProtocol/ticket#2955
Quick polish of what is section.

### CODE REVIEW

File: docs/concepts/01_what_is_kilt.md [hunk 1]
```diff
@@ -4,56 +4,63 @@ title: What is KILT?
...
 
 KILT's main goal is to generate a level playing field for companies to explore new business models related to trust relationships and data sovereignty.
-KILT enables businesses and governments to rely on a common standard which is owned by everyone participating and not by a single company or set thereof.
+KILT enables businesses and governments to rely on a common standard owned by everyone participating and not by a single company or set thereof.
 
 ## The Problem
 
-In the beginning, identity and trust between entities was organized in a fully decentralized way: trust relationships were created directly between individuals based on their own observations.
-Of course this had major drawbacks:
+In the beginning, identity and trust between entities were organized in a fully decentralized way. Individuals created trust relationships directly between them based on their observations.
```
Comment (Dudleyneedham) [inline docs/concepts/01_what_is_kilt.md]:
```suggestion
In the beginning, identity and trust between entities were organized in a fully decentralized way.
Individuals created trust relationships directly between them based on their observations.
```

File: docs/concepts/01_what_is_kilt.md [hunk 2]
```diff
@@ -4,56 +4,63 @@ title: What is KILT?
...
-They no longer simply attest someone's email account, but due to their business model, our personal data (i.e., our identity) is stored and controlled by those same service providers.
+With the invention of the internet, and later of Web 2.0, services evolved and merged into totally centralized solutions including Google, Meta, and X among others.
+They no longer attest to someone's email account, but due to their business model, those same service providers store and control our personal data (i.e., our identity).
 For instance, they could stop allowing us to log into a certain website if they decide to.
-More often than not, companies store the data not only out of necessity, but also for their own business purpose: every time users log into any service, they generate new data points which are then aggregated and sold for advertising purposes.
+More often than not, companies store the data out of necessity and for their own business purpose. Every time users log into any service, they generate new data points which are then aggregated and sold for advertising purposes.
```
Comment (Dudleyneedham) [inline docs/concepts/01_what_is_kilt.md]:
```suggestion
More often than not, companies store the data out of necessity and for their own business purpose.
Every time users log into any service, they generate new data points which are then aggregated and sold for advertising purposes.
```

**Review by Dudleyneedham — APPROVED** [review]
LGTM. Some comments to put the sentences into new lines.

**Comment by ChrisChinchilla** [comment]
@Dudleyneedham Sorry! That still doesn't come naturally…

**Pull request merged**